src/functional/otf/binding/pybind.py

```diff
@@ -149,19 +149,34 @@ def visit_SidConversion(self, sid: SidConversion):
 
 
 def make_parameter(
-    parameter: interface.ScalarParameter | interface.BufferParameter,
+    parameter: interface.ScalarParameter
+    | interface.BufferParameter
+    | interface.ConnectivityParameter,
 ) -> FunctionParameter:
+    if isinstance(parameter, interface.ConnectivityParameter):
+        return FunctionParameter(name=parameter.name, ndim=2, dtype=np.dtype("int32"))
```

Comment:
Maybe let's extract `np.dtype("int32")` into a variable that can be reused in make_argument, to make sure we don't miss changing both, if we have to change them later.